src/ThemeToggleButton.jsx

```diff
@@ -10,6 +10,7 @@ const themeCookie = 'indigo-toggle-dark';
 const themeCookieExpiry = 90; // days
 
 const ThemeToggleButton = ({ intl }) => {
+  const [isDarkThemeEnabled, setIsDarkThemeEnabled] = useState(false);
```

Comment:
Why is there a need for a state? This file already uses themeCookie/cookie to manage the dark theme toggle across sessions. Can't the same be followed for  a11y?